crates/cargo-wdk/src/actions/build/package_task.rs

```diff
@@ -313,9 +313,28 @@ impl<'a> PackageTask<'a> {
             &arch,
             "-c",
             &cat_file_path,
-            "-v",
-            "*",
         ];
+
+        // DriverVer handling:
+        // 1. When allow_stampinf_version_env_override cfg is enabled, allow an external
+        //    override via STAMPINF_VERSION env var. If the env var is absent we fall
+        //    back to auto-generation (-v *).
+        // 2. Otherwise (stable / default builds) always request auto-generation (-v *).
+        cfg_if::cfg_if! {
+            if #[cfg(allow_stampinf_version_env_override)]
+            {
+                if let Ok(version) = std::env::var("STAMPINF_VERSION") && !version.trim().is_empty() {
```

Comment:
imo pull out all env vars to consts (ex. so they can also be used elsewhere like in the log statement)